promise-or-lie.js

```diff
@@ -2,4 +2,4 @@
 
 module.exports = typeof Promise === 'undefined'
 	? require('lie')
-    : global.Promise;
+	: global.Promise;
```

Comment:
This whole file could likely go at this point, since Promises are now a thing everywhere we care about.

Reply:
Yeah, good point. May as well kill it.